frontend/src/pages/Landing/index.css

```diff
@@ -147,6 +147,11 @@ h1.landing {
   background-color: #148439;
 }
 
+.landing .smart-hub--context-menu-button {
+  max-width: 20px;
```

Comment:
I think this is causing the button to look weird:

<img width="91" alt="Screen Shot 2021-03-17 at 9 32 45 AM" src="https://user-images.githubusercontent.com/285842/111476199-5031ec00-8704-11eb-9529-3e529edb7a94.png">